app/bundles/EmailBundle/Tests/Helper/MailHelperTest.php

```diff
@@ -263,6 +263,44 @@ public function testBatchMode(): void
         $this->assertArrayNotHasKey('failures', $errors, var_export($errors, true));
     }
 
+    public function testDuplicateEmail(): void
+    {
+        $this->coreParametersHelper->method('get')->willReturnMap($this->defaultParams);
+
+        $this->contactRepository->method('getLeadOwner')
+            ->willReturnOnConsecutiveCalls(
+                ['email' => 'owner1@owner.com', 'first_name' => 'owner 1', 'last_name' => null, 'signature' => 'owner 1'],
+                ['email' => 'owner2@owner.com', 'first_name' => 'owner 2', 'last_name' => null, 'signature' => 'owner 2'],
```

Comment:
For we do it, I think that tests should be with the same email _owner1@owner.com_

`['email' => 'owner1@owner.com', 'first_name' => 'owner 2', 'last_name' => null, 'signature' => 'owner 2'],`